src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -1906,6 +1906,11 @@ private void WindowClosing(object sender, CancelEventArgs e)
             {
                 isPSSCalledOnViewModelNoCancel = true;
             }
+            //Validates that when Dynamo is being closed we need to close the FileTrust Popup (if is opened)
+            if(fileTrustWarningPopup != null && fileTrustWarningPopup.IsOpen)
+            {
+                fileTrustWarningPopup.IsOpen = false;
+            }
```

Comment:
hi @RobertGlobant20 I understand the intention here, but again since the `TrustWarningPopup` being the only exception here (a specific close), I had to ask why it can't be closed like the other popups? We should have plenty other instances, e.g. incanvas search, notification center, guided tour?

Reply:
@QilongTang I noticed that this code is not needed due that there is already funcionality closing the Popup when Dynamo is closed.
But related to your question, the fileTrustWarningPopup is begin hidden/shown depending if Dynamo window is Active or not, if you are aware about any problem in NotificationCenter or GuidedTour popups please let me know so I can investigate more about it.